publishers/community/slack/slack_layout.py

```diff
@@ -126,7 +126,7 @@ def publish(self, alert, publication):
             'color': self._color(),
             'fields': [
                 {'title': key.capitalize(), 'value': rule_presentation['fields'][key]}
-                for key in rule_presentation['fields'].keys()
+                for key in list(rule_presentation['fields'].keys())
```

Comment:
this one and the ones following I'm actually unsure of though - why the cast when it's being iterated on?